routes/deductions/deductions.controller.js

```diff
@@ -29,6 +29,10 @@ module.exports = function(app) {
     checkSchema(rrspSchema),
     checkErrors('deductions/rrsp'),
     doYesNo('rrspClaim', 'rrspAmount'),
```

Comment:
So, maybe this should be clearer but the second parameter in `doYesNo` actually clears the value if it's there. (because of the repetitive nature of this stuff).

https://github.com/cds-snc/cra-claim-tax-benefits/blob/master/utils/index.js#L158

So we don't need to so this here or the others.